python/ray/tests/test_draining.py

```diff
@@ -104,7 +104,7 @@ def ping(self):
 
     gcs_client = GcsClient(address=ray.get_runtime_context().gcs_address)
 
-    with pytest.raises(ray.exceptions.RpcError):
+    with pytest.raises(ray.exceptions.RaySystemError):
```

Comment:
is there any breaking changes. Ideally this PR shouldn't touch serve code or any tests.

Reply:
This is a fix forward. `drain_node` returns an error from GCS side which should not be considered a RpcError (indicates network issue). Ray serve changes fix bad test fixtures.